src/Native/Unix/System.Native/pal_datetime.h

```diff
@@ -6,8 +6,4 @@
 
 #include "pal_compiler.h"
 
-BEGIN_EXTERN_C
-
 DLLEXPORT int64_t SystemNative_GetSystemTimeAsTicks(void);
-
-END_EXTERN_C
```

Comment:
Why were these removed?

Reply:
Now it is C code, so `extern "C"` is not necessary. In https://github.com/dotnet/corefx/pull/29374 they were C++, so it is needed.

https://github.com/dotnet/corefx/blob/69c2e2072574805589becac0c08590dc5e39b65e/src/Native/Unix/Common/pal_compiler.h#L12-L13
Is a nop for C anyway. Shall I undo this change?

Reply:
Even in that change it was used in .c, e.g. https://github.com/dotnet/corefx/pull/29374/files#diff-b323c460fe656262ae1f2d1e1c49fc29R9.  You might be right that it's unnecessary, but I think it'd be good to be consistent and then they can all be removed en mass when they've been validated to not be needed.